src/components/yelp/yelpReviews.tsx

```diff
@@ -0,0 +1,57 @@
+import { Box } from '@mui/material';
+import React from 'react';
+import { useEffect } from 'react';
+
+const reviewsIds = [
+  'PnTjWZ_jC9f2wY8VnEcRLg',
+  '5Nb2ouXgXh63FsWOjIjTaA',
+  '6cazOQg35n-SwgW8QqgFBQ',
+  'sUgc5QimCys-2K-shv9_2Q',
+  'CwOlA4H6v3VLnsbNW_dgjg',
+  'hBlWEnF7JT8uzGgAOAzh7w',
+  'xlfrONiaMQF-cR3qFq1RIA',
+  'rRocVuVR9wpyQcVr_qFEsQ',
+  '_9fJiQOcj9lMQ00qUs-SfA',
+  'coJc7jwOzhbVGx1vpgYDJg',
+  'RnMw6C5oheiD2LiDio7_jg',
+];
+
+function YelpReviews() {
+  const spansArr = reviewsIds.map((reviewId) => (
+    <div key={reviewId} style={{ padding: '15px', minWidth: '400px' }}>
+      <span
+        className="yelp-review"
+        data-review-id={reviewId}
+        data-hostname="www.yelp.com"
+      ></span>
+    </div>
+  ));
+  useEffect(() => {
+    const scriptTag = document.createElement('script');
```

Comment:
Do we need to do this via a script? Is there a more react way to do it?

Reply:
that I am not sure about. I tried to look up Yelp api, and in order to get access to it, you need to register with yelp and they need to approve your access. 

There is a way to embed reviews by copying their HTML link which included the script to their widgets.js to customize and style the tags. without the script, reviews dont look good. maybe there is a better way to do this.